mlrun/utils/regex.py

```diff
@@ -12,9 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# k8s character limit is for 63 characters
+k8s_character_limit = [r"^.{0,63}$"]
+
 # k8s label value format
 # https://github.com/kubernetes/kubernetes/blob/v1.20.0/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L161
-label_value = [r"^.{0,63}$", r"^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$"]
+label_value = [r"^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$"] + k8s_character_limit
```

Comment:
I would actually change the order, just as a best practice as it will take less time to validate the character limit rather than the other regex. 😅